contracts/src/FeeContract.sol

```diff
@@ -13,18 +13,17 @@ contract FeeContract is Initializable, OwnableUpgradeable, UUPSUpgradeable {
     //
     /// @notice Notify a new deposit
     event Deposit(address indexed user, uint256 amount);
-    event Log(string func, uint256 gas);
 
     /// @notice upgrade event when the proxy updates the implementation it's pointing to
     event Upgrade(address implementation);
+
     // === Constants ===
     //
     /// @notice max amount allowed to be deposited to prevent fat finger errors
     // @TODO confirm this amount with product
+    uint256 public constant maxDepositAmount = 1 ether;
```

Comment:
Can we change this value easily when upgrading? cc @alysiahuggins

Reply:
Corrected. Those were not meant to be constant.